CHANGELOG.md

```diff
@@ -1,239 +1,5 @@
 # Changelog
```

Comment:
Why is this changelog empty now? Shouldn't it list at least the 4.0.0 release?

Reply:
The idea is to only collect changes to the collection here. The change log of the previous role is stored in `UNATTENDED_UPGRADES_CHANGELOG.md`.